main.cpp

```diff
@@ -52,106 +52,114 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: slider
+//  action 1: the slider goes up and down
+slider.moveUpAndDown();
+//  action 2: the slider reutrn to previous position
+slider.undoPreviousPosition();
+//  action 3: the slider return to default position
+slider.defaultPosition();
+  
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: button
+//  action 1: the button extends
+button.extend();
+//  action 2: the button is glow
+button.glow();
+//  action 3: the button is changing his text
+button.changingText();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: speaker
+//  action 1: the speaker make sounds
+speaker.makeSounds();
+//  action 2: the speaker light change color
+speaker.changeLightColor();
+//  action 3: the speaker shut down
+speaker.shutDown();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.